src/extras/amd.js

```diff
@@ -54,7 +54,12 @@ import { errMsg } from '../err-msg.js';
         setters: setters,
         execute: function () {
           var amdResult = amdExec.apply(exports, depModules);
-          if (amdResult !== undefined)
+          if (
+            amdResult !== undefined ||
+            (amdResult === undefined &&
```

Comment:
This part is unnecessary, since the `||` will only happen when the amdResult is undefined